presto-main/src/test/java/com/facebook/presto/operator/scalar/TestVarbinaryFunctions.java

```diff
@@ -138,6 +140,24 @@ public void testFromHex()
     }
 
     @Test
+    public void testToBigEndian64()
+            throws Exception
+    {
+        assertFunction("to_big_endian_64(0)", VARBINARY, new SqlVarbinary(Longs.toByteArray(0)));
```

Comment:
I don't like doing this kind of conversion in tests. Better to just put the exact hex value you expect.

You are assuming that `toByteArray` does big endian. If you were wrong about that, and made the same mistake in the code, this test would pass. However, if this contained the expected hex value, it would be easy to tell if it is correct.